pele/gui/ui/mplwidget.py

```diff
@@ -1,6 +1,6 @@
 from PyQt4 import QtGui, QtCore
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
-from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
+from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
```

Comment:
current version of matplotlib is 1.5. I have 1.3.1 on my computer, which I think is the one that comes with apt-get and I don't get an import error when running the nosetests, so I think we're good